lib/twilio/call.ts

```diff
@@ -1228,6 +1230,7 @@ class Call extends EventEmitter {
         this._mediaHandler.version.getSDP(),
         this.parameters.CallSid,
         this._signalingReconnectToken,
+        this._options.callMessageEvents,
```

Comment:
Let's add unit test to this as well. You can pass callMessageEvents [here](https://github.com/twilio/twilio-voice.js/blob/VBLOCKS-2885/opt-in-call-message/tests/unit/call.ts#L1914). Then perform your assertion [here](https://github.com/twilio/twilio-voice.js/blob/VBLOCKS-2885/opt-in-call-message/tests/unit/call.ts#L1931).